src/components/WelcomeText.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {Text} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+
+const propTypes = {
+    isSmallText: PropTypes.bool,
+};
+
+const defaultProps = {
+    isSmallText: false,
```

Comment:
https://github.com/Expensify/Expensify.cash/blob/efd6a2dd748b7fb180974279f5174e0dba385067/src/components/Header.js#L11-L12 

Maybe we can follow this convention here?